stablehlo/dialect/VhloBytecode.cpp

```diff
@@ -1197,6 +1215,66 @@ void VhloBytecodeInterface::write(TupleV1Type type,
   writer.writeTypes(type.getTypes());
 }
 
+//===----------------------------------------------------------------------===//
+// UniformQuantizedPerAxisV1Type
+//===----------------------------------------------------------------------===//
+
+UniformQuantizedPerAxisV1Type
+VhloBytecodeInterface::readUniformQuantizedPerAxisV1Type(
+    DialectBytecodeReader &reader) const {
+  LOG_READ_CALL;
+  uint64_t flags;
+  Type storageType, expressedType;
+  uint64_t quantizedDimension;
+  int64_t storageTypeMin, storageTypeMax;
+  SmallVector<APFloat> scales;
+  SmallVector<int64_t> zeroPoints;
+  auto readScales = [&]() -> FailureOr<APFloat> {
+    auto valOr =
```

Comment:
How about `return reader.readAPFloatWithKnownSemantics(llvm::APFloat::IEEEdouble());` ?

I believe that returns `FailureOp<APFloat>` as it is?

Reply:
that is right! done.